src/components/postgresConnect.ts

```diff
@@ -397,6 +396,52 @@ class PostgresDB {
     }
   }
 
+  public getUserOrgs = async (userId: number) => {
+    const text = `
+    SELECT organisation.id AS "orgId",
+    user_role as "orgRole",
```

Comment:
not sure what the intended function of user_role is

Reply:
We haven't really discussed how org role would be used, but we will need at least some concept of a company "Owner" to distinguish them from just regular company members. For now it's just a string field and I've put "Owner" in a couple of them, but it's not really doing anything yet.